tests/dataframes_test.py

```diff
@@ -235,7 +321,8 @@ def test_run_query_e2e_run(self):
         columns = dataframes.Columns("privacy_key", "group_key", "value")
         metrics = {
             pipeline_dp.Metrics.COUNT: "count_column",
-            pipeline_dp.Metrics.SUM: None  # it returns default name "sum"
+            pipeline_dp.Metrics.SUM: None,  # it returns default name "sum"
+            pipeline_dp.Metrics.MEAN: None  # it returns default name "sum"
```

Comment:
nit: mean

btw, I had exactly this question what happens if it is not provided (None), should we add it somewhere in documentation in the source class?